cmd/bnsd/x/nft/username/handler.go

```diff
@@ -177,10 +177,10 @@ func (h RemoveChainAddressHandler) Deliver(ctx weave.Context, store weave.KVStor
 
 	actor := nft.FindActor(h.auth, ctx, t, nft.UpdateDetails)
 	if actor == nil {
-		return res, errors.ErrUnauthorizedLegacy()
+		return res, errors.ErrUnauthorized
```

Comment:
I would like a .New() on here just to capture a stack trace.

But I can do that as part of my pr